kuka_lbr_iiwa_support/package.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package>
+  <name>kuka_lbr_iiwa_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA LBR IIWA (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA LBR IIWA manipulators. This currently includes the base model.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in
+      the KUKA data sheets online. 
+      http://www.kuka-robotics.com/res/sps/e6c77545-9030-49b1-93f5-4d17c92173aa_Spez_LBR_iiwa_en.pdf
```

Comment:
Please make this a link. Something like `<a href="http://www.kuka-robotics.com/..">data sheets</a>`.

Or if the datasheet actually has a document nr, use that for the link text.